tests/mcmc/test_integrators.py

```diff
@@ -101,7 +106,13 @@ class IntegratorTest(chex.TestCase):
     @parameterized.parameters(
         itertools.product(
             ["free_fall", "harmonic_oscillator", "planetary_motion"],
-            ["velocity_verlet", "mclachlan", "yoshida"],
+            [
+                "velocity_verlet",
+                "mclachlan",
+                "yoshida",
+                "non_euclidean_leapfrog",
+                "minimal_norm",
```

Comment:
For uniformity, we could rename "minimal_norm" to "non_euclidean_mclachlan", and also add "non_euclidean_yoshida". Jakob (MCLMC author) said that using Yoshida hadn't yet given better results for MCLMC, but that he was in favour of experimenting with it in future.

Reply:
yep sounds good.